cmd/hdfs/mv.go

```diff
@@ -57,14 +57,21 @@ func moveInto(client *hdfs.Client, sources []string, dest string, force bool) {
 }
 
 func moveTo(client *hdfs.Client, source, dest string, force bool) {
+	resp, err := client.Stat(dest)
 	if force {
-		err := client.Remove(dest)
-		if err != nil && !os.IsNotExist(err) {
-			fatal(err)
+		// If dest exists and is directory, remove dest directory
+		if err == nil && resp.IsDir() {
+			if err = client.Remove(dest); err != nil {
+				fatal(err)
+			}
+		}
+	} else {
+		if err == nil {
+			fatal("rename", dest, os.ErrExist)
```

Comment:
Yes, you are right! I changed this line to `os.PathError`, and so I undid the changes in `cmd/hdfs/test/mv.bat`.